src/standalone/Snow/snow_parameterizations.jl

```diff
@@ -477,6 +479,42 @@ function energy_from_T_and_swe(S::FT, T::FT, parameters) where {FT}
 
 end
 
+"""
+    volumetric_energy_flux_falling_snow(atmos, p, parameters)
+
+Returns the volumetric energy fluxof falling snow for a PrescribedAtmosphere,
+approximated as -LH_f0 * _ρ_liq *P_snow. This is per unit volume of liquid water, 
+and P_snow is expressed as the volume flux of liquid water resulting from the snow.
+
+We assume that the sensible heat portion of snow is negligible.
+
+This method can be extended to coupled simulations, where atmos is of type
+CoupledAtmosphere, and the energy flux of the falling snow is passed in the
+cache `p`. In that case, this should specify `atmos::PrescribedAtmosphere`.
+"""
+function volumetric_energy_flux_falling_snow(atmos, p, parameters)
+    _LH_f0 = LP.LH_f0(parameters.earth_param_set)
+    _ρ_liq = LP.ρ_cloud_liq(parameters.earth_param_set)
+    return @lazy @. -_LH_f0 * _ρ_liq * p.drivers.P_snow # per unit vol of liquid water
```

Comment:
I am (perpetually) confused about sign conventions. I trust this is correct, but some explanation in the docstring may be helpful. If fluxes are always defined as positive upward, then P_sno < 0 and the energy flux would be in the same direction as the precipitation flux. The sign here, I suppose, is negative because the latent heat of fusion adds a negative component to the total internal energy. Is that right?

Reply:
yes, exactly! I can define `\rho e  = -_LH_f0 * _ρ_liq` first, and then use that in the return statement. I'll update the docstring to clarify.